app/static/src/app/types/responseTypes.ts

```diff
@@ -21,6 +21,7 @@ export interface AppConfig {
     defaultCode: string[],
     readOnlyCode: boolean,
     stateUploadIntervalMillis?: number | null
+    endTime: number
```

Comment:
This should be optional
```suggestion
    endTime?: number
```